demo-app/build.gradle.kts

```
@@ -2,17 +2,20 @@ import java.io.FileInputStream
 import java.util.Properties
 
 plugins {
-    id("otel.android-app-conventions")
+    id("com.android.application") version "8.4.0"
+    id("org.jetbrains.kotlin.android") version "1.9.23"
 }
 
 val localProperties = Properties()
 localProperties.load(FileInputStream(rootProject.file("local.properties")))
 
 android {
     namespace = "io.opentelemetry.android.demo"
+    compileSdk = 34
```

Based on the discussion below: 
```suggestion
    compileSdk = 34
    targetSdk = 34
```